mobile/lib/app.dart

```diff
@@ -89,17 +91,38 @@ class _EnteAppState extends State<EnteApp> with WidgetsBindingObserver {
     unawaited(HomeWidgetService.instance.initHomeWidget());
 
     final thumbnail = await CollectionsService.instance.getCover(collection);
-    unawaited(
-      routeToPage(
-        context,
-        CollectionPage(
-          CollectionWithThumbnail(
-            collection,
-            thumbnail,
-          ),
+
+    final previousGeneratedId =
+        await hw.HomeWidget.getWidgetData<int>("home_widget_last_img");
+    final res = previousGeneratedId != null
+        ? await FilesDB.instance.getFile(
+            previousGeneratedId,
+          )
+        : null;
+
+    routeToPage(
```

Comment:
Curious, isn't the linter complaining because there's a missing `await`?

Reply:
See the .ignore() at the end.